src/__swaps__/screens/Swap/hooks/useEstimatedGasFee.ts

```diff
@@ -65,7 +121,7 @@ export function useSwapEstimatedGasFee(gasSettings: GasSettings | undefined) {
   useAnimatedReaction(
     () => quote.value,
     (current, previous) => {
-      if (!assetToSell.value || !assetToBuy.value || !current || !previous || 'error' in current) return;
+      if (!assetToSell.value || !assetToBuy.value || !current || 'error' in current) return;
```

Comment:
the previous here make it not update on first quote fetch, it was waiting for a refetch to set the state